src/server/components/auth/__tests__/authenticated-user.test.ts

```diff
@@ -27,20 +27,11 @@ test("isAdministrator evaluation is correct", () => {
 test("getLists query is correct for administrator", async () => {
   await administrator.getLists();
 
-  expect(prisma.listsForDashboard.findMany).toHaveBeenCalledWith({
-    orderBy: [
-      {
-        country: "asc",
-      },
-      {
-        type: "asc",
-      },
-    ],
-  });
+  expect(prisma.listsForDashboard.findMany).toHaveBeenCalledWith({});
 });
 
-test("getLists query is correct for user", async () => {
-  await user.getLists();
+test("should call order by same value that is passed in as argument", async () => {
+  await user.getLists([{ admins: "asc" }]);
```

Comment:
I think technically it should always be called with the default properties (unless it is overridden to desc instead of asc) but fine for this test

Reply:
actually - I don't think the test name matches what you're testing. 

`should call order by same value that is passed in as argument` 
but you can't really test order of props if there's only one key/value 😅